rkt/api_service.go

```diff
@@ -42,7 +42,11 @@ var (
 	cmdAPIService       = &cobra.Command{
 		Use:   `api-service [--listen="localhost:15441"]`,
 		Short: "Run API service (experimental, DO NOT USE IT)",
-		Run:   runWrapper(runAPIService),
+		Long: `The API service listens for gRPC requests on the address and
```

Comment:
Quoting style: Should `Run:` value be quoted as it is here, with backticks, or as the value for `Short:` is quoted, in double quotes?